sdk/python/kfp/cli/run.py

```diff
@@ -58,10 +58,11 @@ def list(ctx, experiment_id, max_size):
 @click.option('-w', '--watch', is_flag=True, default=False,
               help='Watch the run status until it finishes.')
 @click.option('-v', '--version', help='ID of the pipeline version.')
+@click.option('-t', '--timeout', default=-1, help='Wait for a run to complete until timeout in seconds.')
```

Comment:
nit: 0 is probably more common to represent no timeout.

Reply:
Thanks for review! DONE